src/Tribe/Main.php

```diff
@@ -2643,7 +2643,7 @@ public function uglyLink( $type = 'home', $secondary = false ) {
 				case 'day':
 					$eventUrl = add_query_arg( array( 'tribe_event_display' => $type ), $eventUrl );
 					if ( $secondary ) {
-						$eventUrl = add_query_arg( array( 'date' => $secondary ), $eventUrl );
+						$eventUrl = add_query_arg( array( 'eventDate' => $secondary ), $eventUrl );
```

Comment:
Nice catch! Should we be correcting the one in the month case as well?

Reply:
Yep, I've checked and it has the same problem. It usually uses the `tribe-bar-date` though. But one thing for sure is that `date` doesn't work. Will change that for `eventDate` as well.